.azure-pipelines/all.yml

```
@@ -117,6 +117,9 @@ jobs:
     - checkName: redis_sentinel
       displayName: Redis Sentinel
       os: linux
+    - checkName: redisenterprise
+      displayName: RedisEnterprise
```

```suggestion
      displayName: Redis Enterprise
```
CI is failing on this